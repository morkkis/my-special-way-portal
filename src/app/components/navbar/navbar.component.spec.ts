import { NavbarComponent } from './navbar.component';
import { TestBed } from '@angular/core/testing';
import { NO_ERRORS_SCHEMA } from '@angular/core';

describe('dashboard component', () => {
  beforeEach(async () => {
    TestBed.configureTestingModule({
      declarations: [
        NavbarComponent,
      ],
      imports: [],
      schemas: [NO_ERRORS_SCHEMA],
    });
  });

  it('should render component as described in snapshot', () => {
    const fixture = TestBed.createComponent(NavbarComponent);
    expect(fixture).toMatchSnapshot();
  });
});
