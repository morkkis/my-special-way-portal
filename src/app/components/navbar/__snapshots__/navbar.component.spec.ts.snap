// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard component should render component as described in snapshot 1`] = `
<app-navbar>
  <nav
    class="navbar navbar-expand-lg navbar-transparent  navbar-absolute fixed-top"
  >
    
    
    <div
      class="container-fluid"
    >
      
      
      <div
        class="navbar-wrapper"
      >
        
        
        <a
          class="navbar-brand"
          href="#"
        >
          Title to be here
        </a>
        
      
      </div>
      
      
      <button
        class="navbar-toggler"
        mat-raised-button=""
        type="button"
      >
        
        
        <span
          class="sr-only"
        >
          Toggle navigation
        </span>
        
        
        <span
          class="navbar-toggler-icon icon-bar"
        />
        
        
        <span
          class="navbar-toggler-icon icon-bar"
        />
        
        
        <span
          class="navbar-toggler-icon icon-bar"
        />
        
      
      </button>
      
      
      <div
        class="collapse navbar-collapse justify-content-end"
        id="navigation"
      >
        
        
        <form
          class="navbar-form"
        >
          
          
          <div
            class="input-group no-border"
          >
            
            
            <input
              class="form-control"
              placeholder="Search..."
              type="text"
              value=""
            />
            
            
            <button
              class="btn btn-white btn-round btn-just-icon"
              mat-raised-button=""
              type="submit"
            >
              
              
              <i
                class="material-icons"
              >
                search
              </i>
              
              
              <div
                class="ripple-container"
              />
              
            
            </button>
            
          
          </div>
          
        
        </form>
        
        
        <ul
          class="navbar-nav"
        >
          
          
          <li
            class="nav-item"
          >
            
            
            <a
              class="nav-link"
              href="#pablo"
            >
              
              
              <i
                class="material-icons"
              >
                dashboard
              </i>
              
              
              <p>
                
                
                <span
                  class="d-lg-none d-md-block"
                >
                  Stats
                </span>
                
              
              </p>
              
            
            </a>
            
          
          </li>
          
          
          <li
            class="nav-item dropdown"
          >
            
            
            <a
              aria-expanded="false"
              aria-haspopup="true"
              class="nav-link"
              data-toggle="dropdown"
              href="http://example.com"
              id="navbarDropdownMenuLink"
            >
              
              
              <i
                class="material-icons"
              >
                notifications
              </i>
              
              
              <span
                class="notification"
              >
                5
              </span>
              
              
              <p>
                
                
                <span
                  class="d-lg-none d-md-block"
                >
                  Some Actions
                </span>
                
              
              </p>
              
            
            </a>
            
            
            <div
              aria-labelledby="navbarDropdownMenuLink"
              class="dropdown-menu dropdown-menu-right"
            >
              
              
              <a
                class="dropdown-item"
                href="#"
              >
                Mike John responded to your email
              </a>
              
              
              <a
                class="dropdown-item"
                href="#"
              >
                You have 5 new tasks
              </a>
              
              
              <a
                class="dropdown-item"
                href="#"
              >
                You're now friend with Andrew
              </a>
              
              
              <a
                class="dropdown-item"
                href="#"
              >
                Another Notification
              </a>
              
              
              <a
                class="dropdown-item"
                href="#"
              >
                Another One
              </a>
              
            
            </div>
            
          
          </li>
          
          
          <li
            class="nav-item"
          >
            
            
            <a
              class="nav-link"
              href="#pablo"
            >
              
              
              <i
                class="material-icons"
              >
                person
              </i>
              
              
              <p>
                
                
                <span
                  class="d-lg-none d-md-block"
                >
                  Account
                </span>
                
              
              </p>
              
            
            </a>
            
          
          </li>
          
  
          
          <li
            class="nav-item"
          >
            
            
            <a
              class="nav-link"
              href="#pablo"
            >
              
              Hi
              
              
              
              <p>
                <a>
                  Logout
                </a>
              </p>
              
            
            </a>
            
          
          </li>
          
        
        </ul>
        
      
      </div>
      
    
    </div>
    
  
  </nav>
  
  
  
</app-navbar>
`;
